app/controllers/application_controller.rb

```diff
@@ -7,14 +7,22 @@ class ApplicationController < ActionController::Base
 
   helper_method :current_profile, :current_conference, :vapid_public_key
 
+  rescue_from ActionController::InvalidAuthenticityToken,
```

Comment:
You can test this out by going to /user_session/new, deleting your cookies, and submitting the form. Another way is to inspect the DOM and change the CSRF token in the head and in the form you want to submit